openquake/sep/landslide/nowicki_jessee.py

```diff
@@ -0,0 +1,143 @@
+from typing import Union
+import numpy as np
+
+
+landcover_values={
+    "14": 0.91,
+    "20": 0.88,
+    "30": 0.78,
+    "40": 0.68,
+    "50": 0.30,
+    "60": 1.77,
+    "70": 1.71,
+    "90": -1.26,
+    "100": 1.50,
+    "110": 0.68,
+    "120": 1.13,
+    "130": 0.79,
+    "140": 1.03,
+    "150": 0.54,
+    "160": 2.34,
+    "180": 1.19,
+    "190": 0.30,
+    "200": -0.06,
+    "220": -0.18,
+    "230": -1.08
+}
+
+
+lithology_values={
+    "mt": -1.87,
+    "nd": -0.66,
+    "pa": -0.78,
+    "pb": -1.88,
+    "vi": -1.61,
+    "py": -1.05,
+    "sc": -0.95,
+    "sm": -1.36,
+    "ss": -1.92,
+    "su": -1.36,
+    "va": -1.54,
+    "vb": -1.50,
+    "pi": -0.81
+}
+
+
+LANDCOVER_TABLE = {**landcover_values, **{bytes(k, 'utf-8'): v for k, v in landcover_values.items()}}
+LITHOLOGY_TABLE = {**lithology_values, **{bytes(k, 'utf-8'): v for k, v in lithology_values.items()}}
+
+
+def sigmoid(x):
+    return 1.0 / (1.0 + np.exp(-x))
+
+
+def _landslide_spatial_extent(a: float, b: float, c: float, d: float, p: float):
```

Comment:
If the coefficients `a`, `b`, `c`, `d` in this equation are constants, then it would be better to define them inside the function and take only `p` as an input to the function.

Reply:
fixed